lib/trino-record-decoder/src/main/java/io/trino/decoder/raw/RawColumnDecoder.java

```diff
@@ -150,7 +150,8 @@ public RawColumnDecoder(DecoderColumnHandle columnHandle)
                         "Bytes mapping for column '%s' does not match dataFormat '%s'; expected %s bytes but got %s",
                         columnName,
                         fieldType.getSize(),
-                        end.getAsInt() - start);
+                        end.getAsInt() - start,
+                        fieldType.getSize());
```

Comment:
`fieldType.getSize()` is used twice now, is the first occurrence correct? From the message it looks like it should be some date format string, not size.

Reply:
Good catch. Yeah, it's all mixed up...